recipes-kernel/linux/files/patches-5.10/0127-net-ethernet-ti-Fix-buffer-SRAM-overlaps-in-EMAC-mod.patch

```diff
@@ -0,0 +1,43 @@
+From 5f5b824f05de704f989d2f12a5cbbe7078e05356 Mon Sep 17 00:00:00 2001
+From: Vignesh Raghavendra <vigneshr@ti.com>
+Date: Mon, 6 Dec 2021 14:12:07 +0530
+Subject: [PATCH 127/145] net: ethernet: ti: Fix buffer SRAM overlaps in EMAC
+ mode
+
+Buffer allocation was supposed to follow below layout as per comment in
+the prueth_emac_buffer_setup()
+
+[1] |BPOOL0 (64K) |BPOOL1 (64K)|RX_CTX0 (32K)|RX_CTX1 (32K)|
+
+But at the moment tracing prueth_emac_buffer_setup() shows below layout:
+
+[2] |BPOOL0 (64K) |BPOOL1 (64K)|RX_CTX0 (32K)| EMPTY (64K) |RX_CTX1 (32K)|
+
+Additional empty 64K is not required and is a bug. This causes ICSSG0
+SRAM reservations to overlap with ICSSG1 and so on. Fix this by dropping
+unintended empty 64K hole and this reverting to layout [1].
```

Comment:
Huh, that could explain an issue we see on SR1.0 so far only, that random packets are sometimes sent over one NIC, sometimes even packets from the other one...

Reply:
...but this one is targeting sr2 code paths only. Too bad.